src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -111,6 +111,11 @@ public void setPerson(Person target, Person editedPerson) {
         addressBook.setPerson(target, editedPerson);
     }
 
+    @Override
+    public void sortByPin() {
```

Comment:
I like that this method is implemented to allow adherence to the Law of Demeter!